buildが失敗する

GitHub ActionsでNext.jsビルド時に以下のエラー

```error
Run yarn next build
yarn run v1.22.22
$ /home/runner/work/sc4pe_pre2/sc4pe_pre2/node_modules/.bin/next build
  ▲ Next.js 14.2.5

   Creating an optimized production build ...
 ✓ Compiled successfully
   Linting and checking validity of types ...
Failed to compile.

./src/app/page.tsx:5:24
Type error: Could not find a declaration file for module 'three'. '/home/runner/work/sc4pe_pre2/sc4pe_pre2/node_modules/three/build/three.module.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/three` if it exists or add a new declaration (.d.ts) file containing `declare module 'three';`

  3 | import Head from 'next/head';
  4 | import { useEffect, useRef } from 'react';
> 5 | import * as THREE from 'three';
    |                        ^
  6 |
  7 | export default function Home() {
  8 |   const mountRef = useRef<HTMLDivElement>(null);
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/package.json
+++ b/package.json
@@ -18,6 +18,7 @@
     "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
+    "@types/three": "^0.166.0",
     "eslint": "^8",
     "eslint-config-next": "14.2.5",
     "typescript": "^5"

